### ISSUE TITLE
Add url parameter for collapsing sidebar

### ISSUE THREAD

**Issue by jeaster12** [storybookjs/storybook#6506]
**Is your feature request related to a problem? Please describe.**
Currently collapsing the sidebar is only available globally in the options, but not as a URL parameter. 

For instance we can embed the story in a buttons section of a docs with all the correct options and knobs selected and present in the URL. However the sidebar remains open - if we collapse it in the config - it ruins the usability of the storybook standalone not embedded in an app. 

**Describe the solution you'd like**
Ideally the selected sidebar state could be passed in the url 
designsystem.com/?path=/story/breadcrumb&sidebar-isCollapsed=true

**Describe alternatives you've considered**

- Using just the iframe as an embed: lacks the excellent knob and full canvas functionality
- Collapse the sidebar in config: makes the actual app unusable as a standalone

**Comment by divbhasin** [comment]
@jeaster12 @shilman I can work on this feature. I was thinking that the user can probably just toggle the menu themselves if we present the static storybook but that is a few extra steps which is bad for usability if the intent is to just showcase a component.

**Comment by divbhasin** [comment]
I was going through the codebase and saw that we already support something like this. In https://github.com/storybooks/storybook/blob/next/lib/api/src/modules/url.ts, there is support for 'nav' parameter. 'nav=0' hides the sidebar, and 'nav=1' shows it (the default).

**Comment by jeaster12** [comment]
@divbhasin fantastic yes this actually works perfect! 

@shilman shall we close this? or are there additional needs or opportunities?

**Comment by shilman** [comment]
Awesome, closing! 🚀

**Comment by Xeno007** [comment]
I am unable to use this url parameter, is there some type of setup that needs to be done in order to use the nav=0 parameter?

We are trying to import storybook into Zeroheight, however with the import or just in browser adding the url parameter it just resets the url to the initial story url.

**Comment by Xeno007** [comment]
@shilman Yes I am trying to embed the entire Storybook UI without the side menu. I have taken a look at setting the options parameters in our config file, however then if we navigate to our storybook site the problem is that the side menu is hidden there as well

**Comment by IsurusPrime** [comment]
I too would like to embed the page in an iframe and include the controls for the knobs at least. Anything else is a bonus.

I tried adding "&nav=0" to my url like so: 
`<iframe width="100%" height="720px" src="https://agrium.github.io/nutrien-bonsai/?path=/story/banner--default&nav=0" allowfullscreen></iframe>` 

but that doesn't hide the nav when the iframe loads.

**Comment by JamesIves** [comment]
Is there any updates on when this will be made available in 8.x?
